We should not depend on pytest-runner unconditionally, there are situations where we don't run tests like when bootstrapping new arches

diff --git a/setup.py b/setup.py
index 1ba1dcd..38cb78a 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,6 @@ with io.open(path.join(this_directory, 'README.txt'), encoding='utf-8') as f:
 setup (name = "managesieve",
        version='0.6',
        python_requires='>=2.7',
-       setup_requires=["pytest-runner"],
        tests_require=["pytest"],
        description = description,
        long_description = long_description,
